front/components/aroundMe/fetchPoisCoords.component.tsx

```diff
@@ -50,16 +52,21 @@ const FetchPoisCoords: React.FC<Props> = ({
       const savedTypeFilter: string[] = await StorageUtils.getObjectValue(
         StorageKeysConstants.cartoFilterKey
       );
-      const variables = {
-        lat1: topLeftPoint.latitude,
-        lat2: bottomRightPoint.latitude,
-        long1: topLeftPoint.longitude,
-        long2: bottomRightPoint.longitude,
-        types: savedTypeFilter,
-      };
-      getPoisByGpsCoords({
-        variables,
-      });
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+      if (!savedTypeFilter || savedTypeFilter.length === 0) {
```

Comment:
Je ferais un leave-early ici pour désindenter le cas nominal (else).

```suggestion
      if (!savedTypeFilter || savedTypeFilter.length === 0) return chooseFilterMessage()
```